tensorboard/webapp/runs/views/runs_table/runs_table_component.scss

```diff
@@ -51,38 +46,35 @@ $_table-content-row-min-height: 43px;
   }
 }
 
-table {
-  width: 100%;
+$_border-color: #0000001f;
 
-  th,
-  td {
-    font-size: 13px;
-    padding: 5px;
-
-    &:not(.name) {
-      text-align: center;
-    }
-
-    &:first-of-type {
-      padding-left: 24px;
-    }
-
-    &:last-of-type {
-      padding-right: 24px;
-    }
-  }
-
-  th {
+[role='grid'] {
+  .header {
     color: mat-color($tb-foreground, text);
-  }
-
-  tr:not(.select-all) th {
     white-space: nowrap;
   }
 
-  .mat-header-row,
-  .mat-row {
+  .header,
+  .run-row {
+    contain: strict;
+    display: table-row;
```

Comment:
No action necessary, just wanted to comment that I'm rather unfamiliar with when to use the `display: table-*` CSS values and the details on how they are rendered/layout-ed differently from `display: block`.  I'd be happy to learn what benefits of using these over block/flex are.

Reply:
If you see current TensorBoard and see their role and their computed `display` values, you will get some sense of what it should be. This change basically implements `table` without using `table`.